### PR TITLE
cf-remote: Added 'scp', 'sudo', and 'run' commands 

### PR DESCRIPTION
**Pull request by olehermanse**
Wait for #3512

### CODE REVIEW

**Comment by olehermanse** [comment]
@craigcomstock @vpodzime Please review.

File: contrib/cf-remote/cf_remote/commands.py [hunk 1]
```diff
@@ -15,6 +15,21 @@ def info(hosts, users=None):
...
 
 
+def run(hosts, command, users=None, sudo=False):
+    assert hosts
+    for host in hosts:
+        lines = run_command(host=host, command=command, users=users, sudo=sudo)
+        if lines is None:
+            print("Failed on host: '{}'".format(host))
+            continue
+        for line in lines.split("\n"):
```
Comment (vpodzime) [inline contrib/cf-remote/cf_remote/commands.py]: `lines.splitlines()`

Comment (olehermanse) [reply contrib/cf-remote/cf_remote/commands.py]: Can be fixed in follow-up.

**Comment by olehermanse** [comment]
Merging to move this along: https://github.com/cfengine/core/pull/3520

**Pull request merged**